src/components/Profile/components/SchedulePanel/components/EditDescriptionDialog/index.jsx

```diff
@@ -50,11 +50,13 @@ export default class EditDescriptionDialog extends Component {
     return (
       <Dialog open={this.props.editDescriptionOpen} keepMounted fullWidth="true" maxWidth="xs">
         <div className={styles.desc_tile}>
-          <DialogTitle className={styles.desc_title}>Edit schedule description</DialogTitle>
+          <DialogTitle className={styles.desc_title}>Edit Public Office Hours Note</DialogTitle>
```

Comment:
I'm confused about this, don't we already have a spot for displaying `facstaff` office hours?

Reply:
We are moving the current office hour display to the schedule panel